tests/testthat/test-standalone-forcats.R

```diff
@@ -54,3 +54,20 @@ test_that("fct_relevel() works", {
   f_new <- fct_relevel(f, "b", "a", "d")  # "d" is unobserved
   expect_equal(levels(f_new), c("b", "a", "d", "c"))  # "d" should be added as a level, even though not observed
 })
+
+test_that("fct_* function behaviour when input is not a factor ", {
+  f <- factor(c("b", "b", "a", "c", "c", "c"))
+  c <- as.character(c("b", "b", "a", "c", "c", "c"))
+  expect_equal(forcats::fct_infreq(f), fct_infreq(c))
+  expect_equal(forcats::fct_inorder(f), fct_inorder(c))
+  expect_equal(forcats::fct_rev(f), fct_rev(c))
+  expect_equal(forcats::fct_relevel(fct_relevel(f, "b", "a")), fct_relevel(fct_relevel(c, "b", "a")))
```

Comment:
I think this last one needs the `forcats::` prefix?

Reply:
or the second one?